### ISSUE TITLE
Julia: Enable and Disable Compiled mode for debugger. UI confusing

### ISSUE THREAD

**Issue by JaapHaitsma** [julia-vscode/julia-vscode#2683]
![image](https://user-images.githubusercontent.com/905295/150243807-57dca12f-85f6-47d1-8257-373ffb1a7a05.png)
Apparently when the circle is empty compiled code is enabled and when it's red it is disabled. This is quite confusing. A full circle usually means that something is enabled.

**Comment by pfitzseb** [comment]
Yeah, that's fair. The UI is supposed to tell you whether breakpoints are enabled (red breakpoint icon means that breakpoints work, hollow and grey icon means that they don't, just like in the rest of the editor).

Should probably just be text after all...

**Comment by heliosdrm** [comment]
Not exactly: it took me a few tests to understand it, but although grey breakpoint icon does mean that no breakpoint works, red breakpoint icon means that breakpoints work *except in modules/functions that are included in the "Compiled" list* (for obvious reasons).

So, I'd suggest rewrite the hints so that they tell:
* When the button is red: "Disable Breakpoints for the Debugger"
* When the button is grey: "Enable Breakpoints of Interpreted Code for the Debugger"

(If the asymmetry of that suggestion is not welcome, "of Interpreted Code" in the second label can be left out.)

I think that those messages represent more clearly how it works, and have the advantage of meaning red=enabled, grey=disabled.

**Comment by DNF2** [comment]
The current state is very confusing. Right now the debugger is not working for me, and I cannot tell whether red dot means 'enabled' or 'disabled'. I suggest that the popup-text should describe the current state, and not the result of an action, roughly like this:

Green (not red, red is confusing): "Compiled mode for debugger: Enabled."
Grey: "Compiled mode for debugger: Disabled."

Or vice versa, I don't get what it means now.
